apps/site/src/content/config.ts

```diff
@@ -6,13 +6,33 @@ const blogCollection = defineCollection({
 	schema: z.object({
 		title: z.string(),
 		draft: z.boolean(),
+		publishDate: z.coerce.date(),
```

Comment:
This is unsafe. If `publishDate` is `null` would still parse it as `true.` [Zod's documentation](https://zod.dev/?id=you-can-use-pipe-to-fix-common-issues-with-zcoerce)
```suggestion
		publishDate: z.union([z.number(), z.string(), z.date()]).pipe(z.coerce.date()),
```